File: SourceCode/Debug Helpers/open.fda.gov/ApiDalc/DataObjects/DrugEventList.vb
<Obsolete>
Public Class DrugEventList

    Public Property Events As New List(Of DrugEventItem)

    Public Sub New(jsonString As String)

        Dim x As Dictionary(Of String, Object) = Newtonsoft.Json.JsonConvert.DeserializeObject(Of Dictionary(Of String, Object))(jsonString)
        Dim listing As List(Of Dictionary(Of String, Object)) = Newtonsoft.Json.JsonConvert.DeserializeObject(Of List(Of Dictionary(Of String, Object)))(x("results").ToString)

        For Each item In listing

            Events.Add(DrugEventItem.Fill(item))

        Next

    End Sub

End Class

File: SourceCode/AspCoreVersion/src/ShopAware.Core/DataObjects/DrugEventList.cs
using System;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace ShopAware.Core
{
    [Obsolete]
    public class DrugEventList
    {
        #region Properties

        public List<DrugEventItem> Events { get; set; }

        #endregion

        #region Constructors

        public DrugEventList(string jsonString)
        {
            // VBConversions Note: Non-static class variable initialization is below.  Class variables cannot be initially assigned non-static values in C#.
            Events = new List<DrugEventItem>();

            var x = JsonConvert.DeserializeObject<Dictionary<string, object>>(jsonString);
            var listing = JsonConvert.DeserializeObject<List<Dictionary<string, object>>>(x["results"].ToString());

            foreach (var item in listing)
            {
                Events.Add(DrugEventItem.Fill(item));
            }
        }

        #endregion
    }
}